internal/compiler/emitter.go

```diff
@@ -298,7 +372,17 @@ type outputPaths struct {
 	buildInfoPath       string
 }
 
-func getOutputPathsFor(sourceFile *ast.SourceFile, host printer.EmitHost, forceDtsEmit bool) *outputPaths {
+// DeclarationFilePath implements declarations.OutputPaths.
+func (o *outputPaths) DeclarationFilePath() string {
+	return o.declarationFilePath
+}
+
+// JsFilePath implements declarations.OutputPaths.
+func (o *outputPaths) JsFilePath() string {
```

Comment:
FWIW it's more canonical in Go to write `JSFilePath`; I think we've mostly done that elsewhere?

Reply:
But, we also write `Json` in places still so, shrug